readme.md

```diff
@@ -2,10 +2,10 @@
 # AMP Plugin for WordPress
 
 ![Banner](wp-assets/banner-1544x500.png)
-Enable AMP on your WordPress site, the WordPress way.
+Official AMP plugin, supported and maintained by AMP. Formerly “Accelerated Mobile Pages”, now for mobile and desktop. Use for website and stories!
```

Comment:
What about... "Official AMP plugin, supported by the AMP team. Formerly "Accelerated Mobile Pages", AMP enables great experiences across mobile, desktop & stories."  ?